components/match2/wikis/apexlegends/match_summary.lua

```diff
@@ -44,6 +44,14 @@ local TROPHY_COLOR = {
 	'icon--copper',
 }
 
+local STATUS_ICONS = {
+	up = 'fas fa-chevron-double-up',
+	stayup = 'fas fa-chevron-up',
+	stay = 'fas fa-equals',
+	staydown = 'fas fa-chevron-down',
+	down = 'fas fa-skull',
+}
```

Comment:
would we want to move them to the icon module?

Reply:
Leaving it for later, but if they become standard, then yes